design/specs/table-import.md

```diff
@@ -14,153 +14,175 @@ Adding a new table from a file import allows users to input data into Mathesar w
 While data file import is a baseline feature for most applications in this category, additional functionality will be implemented in the future to enhance the data import process, such as automatic type detection, data cleansing, error detection, and others.
 
 # User Experience
-## Impact
-This feature will improve the user experience for users who need to input data into Mathesar by reducing the required effort when populating tables.
 
-This feature will also support the experience of first-time users, as it is expected by users of similar applications and would cause dissatisfaction if missing.
+## Navigating to the Data Import page
 
-## User Flow
+![](https://share.balsamiq.com/c/p3dqSXr66qa6EnPyykqpyZ.png)
 
-![](/assets/design/specs/table-import/Z3ngzR9.png)
+Data import navigation link will be found on the active schema page. Since we are only allowing users to import tables for now, positioning it with the existing tables/ active schema view made sense. 
```

Comment:
It would be enough to mention that user can access data import from the Schema page, since there will be a separate spec to address the design and layout for this page.